### ISSUE TITLE
LAWP and PTR have extremely low accuracy.

### ISSUE THREAD

**Issue by FenodyreeAv** [PolarisSS13/Polaris#6374]
#### Brief description of the issue
The PTR got a huge nerf, with the removal of goggles working through scopes and binoculars.
Both sniper rifles also have very low accuracy, even scoped in.

The PTR hits one shot in every three on average.
The LWAP hits roughly one in four.

In a oneshot, and four shot weapon respectively, this means that unless you are VERY lucky, you won't be killing anyone.

#### What you expected to happen
A sniper rifle to hit while scoped, most of the time.

#### What actually happened
Many narrow misses.

#### Steps to reproduce
Set the burstfire number on the LWAP to 100, and shoot a dummy. Then check the burn damage.

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

**Comment by zekesulastin** [comment]
[s]Remove snipers/[/s]

**Comment by FenodyreeAv** [comment]
I agree, it should be one or the other. But at the moment it's neither ):

**Comment by SpadesNeil** [comment]
The effectiveness of snipers is drastically reduced by the restriction of how corridors cut off line of sight prematurely. Snipers are only truly effective in the open, such as open space, or outdoors.
